package.json

```diff
@@ -219,7 +221,7 @@
 		"build-devdocs:index": "cross-env-shell NODE_PATH=$NODE_PATH:server:client:. npm run -s build-devdocs:index:_env",
 		"build-devdocs:index:_env": "node server/devdocs/bin/generate-devdocs-index",
 		"build-docker": "node bin/build-docker.js",
-		"build-static": "ncp static public",
+		"build-static": "ncp packages/material-design-icons/svg-sprite public/images/material-design-icons && ncp static public",
```

Comment:
this is failing after a `clean`. probably need to copy static into public first, then copy in the icons?

Reply:
This is removed after the switch to using file-loader in cb90ef8.